specification/paths/Webhooks-webhook_id.json

```diff
@@ -1,7 +1,7 @@
 {
   "get": {
     "tags": [
-      "Webhooks"
+      "⚠ Webhooks"
```

Comment:
Should hooks not be removed completely?

Reply:
Removing them completely right now will cause the API tests to fail.